src/components/shared/ScreenReaderContent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
This component already exists in web-slab. It's just with the other components, not in `shared`, so I can see why that was confusing